service/src/main/java/org/cbioportal/service/impl/DiscreteCopyNumberServiceImpl.java

```diff
@@ -120,14 +120,14 @@ public List<DiscreteCopyNumberData> getDiscreteCopyNumbersInMultipleMolecularPro
         List<CNA> cnas = geneQueries.stream().map(q -> q.getAlterations()).flatMap(List::stream).collect(Collectors.toList());
         if (cnas.isEmpty())
             return Collections.emptyList();
-        
+
         if (isHomdelOrAmpOnlyCna(cnas)) {
             return discreteCopyNumberRepository.getDiscreteCopyNumbersInMultipleMolecularProfilesByGeneQueries(molecularProfileIds,
                 sampleIds, geneQueries, projection);
         }
-        
+
         return molecularDataService.getMolecularDataInMultipleMolecularProfilesByGeneQueries(molecularProfileIds, sampleIds,
-            geneQueries, projection).stream()
+                geneQueries, projection).stream()
```

Comment:
Please correct indentation (make consistent).

Reply:
Fixed in [9f7f6a5](https://github.com/BasLee/cbioportal/pull/1/commits/9f7f6a57df2238791075041f36851b087cdd415f)